src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -147,29 +147,5 @@ def run(input: str, config: str, rules: str, rejects: str, output: str):
     with open(str(Path(output.replace("lexical", "lexical-2"))), "w") as f:
         filter_file(input=str(Path(output)), output=f, **kwargs)
 
-
-def iri_to_curie(item):
```

Comment:
Oh ok good, we're finally getting rid of this. I see it had no usages as well.

Reply:
Yes, that's right.